### DISCUSSION TITLE
Canvas playground

### DISCUSSION THREAD

**Discussion by guiireal** [kyranet/canvas-constructor#515] [Ideas]
I really like using the canvas-constructor lib, but I would like to know if there is an online playground where you use the components, build the figures and see the result in real time (perhaps even an online drawer tool that manipulates the x and y directions). I believe this would speed up development.

**Comment by viniciusgdr** [comment]
It would be an interesting idea!

**Comment by kyranet** [comment]
This is something I wanted to do for a long while, but I'm not knowledgeable enough on how code playgrounds work in browsers 😅. That being said, if you do know how to do such thing, I can build one on my personal website. 👀
